ocaml/doc/jsapi.ml

```diff
@@ -83,14 +87,39 @@ let generate_files destdir =
       "{'cls': '" ^ obj.name ^ "', 'obj_changes': " ^ Jsonrpc.to_string (rpc_of_changes_t obj_changes) ^ ", 'field_changes': " ^ Jsonrpc.to_string (rpc_of_changes_t field_changes) ^ ", 'msg_changes': " ^ Jsonrpc.to_string (rpc_of_changes_t msg_changes) ^ "}"
     in
     let release_info = String.concat ", " (List.map search_obj objs) in
-    let fname = rel ^ ".json" in
+    let fname = (code_name_of_release rel) ^ ".json" in
     Stdext.Unixext.write_string_to_file (Filename.concat api_dir fname) ("release_info = [" ^ release_info ^ "]")
   in
   List.iter changes_in_release release_order;
-  let release_list = String.concat ", " (List.map (fun s -> "'" ^ s ^ "'") release_order) in
+  let release_list = String.concat ", " (List.map (fun s -> "'" ^ (code_name_of_release s) ^ "'") release_order) in
   Stdext.Unixext.write_string_to_file (Filename.concat api_dir "releases.json") ("releases = [" ^ release_list ^ "]")
 
+let json_releases =
+  let json_of_rel x = `O [
+    "code_name", `String (code_name_of_release x);
+    "version_major", `Float (float_of_int x.version_major);
+    "version_minor", `Float (float_of_int x.version_minor);
+    "branding", `String x.branding;
+    "xs_branding", `String x.xs_branding;
+    ]
+  in
+  `O [ "releases", `A (List.map json_of_rel release_order) ]
+
+let render_template template_file json output_file =
+  let lines_read = Stdext.Unixext.read_lines template_file in
+  let templ = Mustache.of_string (String.concat "\n" lines_read) in
```

Comment:
This could be
```ocaml
let templ =
    Stdext.Unixext.string_of_file template_file
    |> Mustache.of_string
in
```
This writes directly to the string, will avoid creating a list and then concatenating it.